akka-docs/src/main/paradox/java/stream/stream-error.md

```diff
@@ -1,76 +1,2 @@
-
-*ZipWith*, *GraphStage* junction, *ActorPublisher* source and *ActorSubscriber* sink
-components do not honour the supervision strategy attribute yet.
-
-@@@
-
-## Supervision Strategies
-
-There are three ways to handle exceptions from application code:
-
- * `Stop` - The stream is completed with failure.
- * `Resume` - The element is dropped and the stream continues.
- * `Restart` - The element is dropped and the stream continues after restarting the stage.
-Restarting a stage means that any accumulated state is cleared. This is typically
-performed by creating a new instance of the stage.
-
-By default the stopping strategy is used for all exceptions, i.e. the stream will be completed with
-failure when an exception is thrown.
-
-@@snip [FlowErrorDocTest.java]($code$/java/jdocs/stream/FlowErrorDocTest.java) { #stop }
-
-The default supervision strategy for a stream can be defined on the settings of the materializer.
-
-@@snip [FlowErrorDocTest.java]($code$/java/jdocs/stream/FlowErrorDocTest.java) { #resume }
-
-Here you can see that all `ArithmeticException` will resume the processing, i.e. the
-elements that cause the division by zero are effectively dropped.
-
-@@@ note
-
-Be aware that dropping elements may result in deadlocks in graphs with
-cycles, as explained in @ref:[Graph cycles, liveness and deadlocks](stream-graphs.md#graph-cycles).
-
-@@@
-
-The supervision strategy can also be defined for all operators of a flow.
-
-@@snip [FlowErrorDocTest.java]($code$/java/jdocs/stream/FlowErrorDocTest.java) { #resume-section }
-
-`Restart` works in a similar way as `Resume` with the addition that accumulated state,
-if any, of the failing processing stage will be reset.
-
-@@snip [FlowErrorDocTest.java]($code$/java/jdocs/stream/FlowErrorDocTest.java) { #restart-section }
-
-## Errors from mapAsync
-
-Stream supervision can also be applied to the futures of `mapAsync`.
-
-Let's say that we use an external service to lookup email addresses and we would like to
-discard those that cannot be found.
-
-We start with the tweet stream of authors:
-
-@@snip [IntegrationDocTest.java]($code$/java/jdocs/stream/IntegrationDocTest.java) { #tweet-authors }
-
-Assume that we can lookup their email address using:
-
-@@snip [IntegrationDocTest.java]($code$/java/jdocs/stream/IntegrationDocTest.java) { #email-address-lookup2 }
-
-The `CompletionStage` is completed normally if the email is not found.
-
-Transforming the stream of authors to a stream of email addresses by using the `lookupEmail`
-service can be done with `mapAsync` and we use `Supervision.getResumingDecider` to drop
-unknown email addresses:
-
-@@snip [IntegrationDocTest.java]($code$/java/jdocs/stream/IntegrationDocTest.java) { #email-addresses-mapAsync-supervision }
-
-If we would not use `Resume` the default stopping strategy would complete the stream
-with failure on the first `CompletionStage` that was completed exceptionally.
\ No newline at end of file
+../../scala/stream/stream-error.md
```

Comment:
This is not a symlink but just a file. I've added some documentation on how to make symlinks to https://github.com/akka/akka/issues/23052#issuecomment-305107157 - you can test if it works with 'sbt paradox'.